vscode/src/common.ts

```diff
@@ -3,6 +3,7 @@
 
 export const qsharpLanguageId = "qsharp";
 export const qsharpExtensionId = "qsharp-vscode";
+export const QsLibraryUriScheme = "qsharp-source-request";
```

Comment:
```suggestion
export const qsharpLibraryUriScheme = "qsharp-library-source";
```

Since you said you were looking for naming suggestions :)